controllers/nnf_workflow_controller.go

```diff
@@ -998,13 +1057,14 @@ func (r *NnfWorkflowReconciler) setupNnfAccessForServers(ctx context.Context, st
 		func() error {
 			dwsv1alpha1.AddWorkflowLabels(access, workflow)
 			dwsv1alpha1.AddOwnerLabels(access, workflow)
-			addDirectiveIndexLabel(access, directiveIdx)
+			addDirectiveIndexLabel(access, index)
 			addTeardownStateLabel(access, teardownState)
 
 			access.Spec = nnfv1alpha1.NnfAccessSpec{
 				DesiredState:    "mounted",
 				TeardownState:   teardownState,
 				Target:          "all",
+				MountPath:       buildMountPath(workflow, params["name"], params["command"]),
```

Comment:
It's either MountPath, or MountPathPrefix.  When would it make sense to have both in the same resource?  (I can never read these two fields without having to pause a moment and remind myself about why they are different, why we have both fields...you're not making this any easier.)